integrations/standalone/tests/mock/browser.spec.ts

```diff
@@ -23,36 +23,45 @@ test.describe('Script browser', () => {
     const task = inscriptionView.accordion('Task');
     await task.toggle();
 
+    const category = task.macroInput('Category');
     await assertCodeHidden(page);
-    await page.getByLabel('Category').click();
+    await category.focus();
     await assertCodeVisible(page);
 
-    await CodeEditorUtil.fill(page, 'test 123 zag');
-    await page.getByRole('code').dblclick();
+    await page.keyboard.type('test 123 zag');
```

Comment:
category.fill

Reply:
fill does make a focus, clear, type and blur. I only want to write...